### DISCUSSION TITLE
Add tracked items to Captcha page

### DISCUSSION THREAD

**Discussion by jensenbox** [djangopackages/djangopackages#654] [Grids]
For https://djangopackages.org/grids/g/captcha/

Add the following tracked data:
- Rate Limiting
- Audio
- Image

**Comment by ryancheley** [comment]
@jensenbox is this a request to add extra meta data about the package in the `captcha` grid?

**Reply by jensenbox** [reply]
Yes, sorry for being so terse on the original request.

It would be nice to see those fields on each package.
